### ISSUE TITLE
img alt should be null, rather than missing aria-label

### ISSUE THREAD

**Issue by simondate** [adaptlearning/adapt_framework#3282]
### Subject of the issue/enhancement/features

One of my clients has partnered with an accessibility testing company to test their Adapt content.

They've noticed that when an image doesn't have any aria-label, in this case intentionally as it's a decorative image, it isn't applied. They are suggesting that it should instead be set as a null value.

See here - https://www.w3.org/WAI/WCAG21/Techniques/html/H67

### Your environment
* version (AT/Framework) FW v5.15.2
* which browser and its version Windows 10 – Chrome 97.0.4692.71
* device(s) + operating system(s)

**Comment by oliverfoster** [comment]
Hi Simon.

### alt attribute
The `alt` attribute is an html attribute for just the `img` tag. When given a value it is a way of indicating to the screen reader that it should read the `alt` attribute value as the "accessible name" of the `img` tag. [H67](https://www.w3.org/WAI/WCAG21/Techniques/html/H67) 
A null string `img` `alt` attribute simply hides the image from the screen reader. [H37](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)

### aria equivalent
We use `aria-label` as a general aria specification way of providing an "accessible name" to any element, not just the `img` tag. [ARIA6](https://www.w3.org/TR/WCAG20-TECHS/ARIA6.html)
We use `aria-hidden` as a general way of hidding any element. [aria-hidden](https://www.w3.org/WAI/standards-guidelines/act/rules/aria-hidden-no-focusable-content-6cfa84/).

### "accessible name" calculation
You can see how the "accessible name" is calculated here. [4. Name and Description](https://www.w3.org/TR/accname-1.2/#mapping_additional_nd)

### alt or aria
There are different interpretations to whether `alt` is required, the references below are some conflicting opinions. However, as can be seen from the "accessible name" calculation, specifying `aria-label` or `aria-labelledby` takes precendence over `alt` and so the issue is almost rendered moot by real world application.

[Is the Alt Attribute Dead?](https://www.levelaccess.com/is-the-alt-attribute-dead/)
[Can aria attributes be used instead of an alt attribute to label non-text content?](https://sitemorse.com/blog-article/9154/can-aria-attributes-be-used-instead-alt-attribute/) (This is regarding WCAG 2.0 not 2.1)
[1.1.1 Non-text Content Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=111#qr-text-equiv-all)
[F39](https://www.w3.org/WAI/WCAG21/Techniques/failures/F39.html)
[F65](https://www.w3.org/WAI/WCAG21/Techniques/failures/F65.html)
[Screen reader support for hidden content](https://stevefaulkner.github.io/HTML5accessibility/tests/hidden-2016.html)
[img with aria-label screen reader compability](https://www.powermapper.com/tests/screen-readers/labelling/img-aria-label/)

**Comment by oliverfoster** [comment]
That being said, if you find specific examples og img tags which don't have `aria-hidden` and with `aria-label`, please open a specific issue on the component with reference to the img tag in question.
